### ISSUE TITLE
[Bug]: An input widget without a label is not draggable on the canvas

### ISSUE THREAD

**Issue by infinitetrooper** [appsmithorg/appsmith#19008]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When I try to select and drag an input widget without a label, the drag functionality doesn't work. I can see the palm icon and selection but, the widget feels locked. See the video below.

### Steps To Reproduce

1. Add an input widget to the canvas.
2. Clear the label section of the input widget.
3. Try to select and drag the widget.

### Issue video log

https://www.loom.com/share/f8ffe769ae74425e8f46b1a1a0ef3fc4

### Version

Appsmith v1.8.13-SNAPSHOT

**Comment by dilippitchika** [comment]
Yeah, this is not draggable because it's interactive. It's a problem we are aware of.

**Comment by infinitetrooper** [comment]
@pranavkanade  - I tried it and it's draggable from the name tag. This is a good work around. A little inconsistent as you can drag most widgets from anywhere.

**Comment by ramsaptami** [comment]
Fixed in current anvil DP
